tests/shared/core/test_slot_mappings.py

```diff
@@ -167,3 +167,52 @@ def test_slot_mappings_check_mapping_validity_from_intent():
         mapping=mappings_for_slot[0],
         domain=domain,
     )
+
+
+@pytest.mark.parametrize(
+    "intent, expected",
+    [
+        (["goodbye", "mood_great", "greet"], True),
+        (["a", "b", "c"], False),
+        ([], True),
+        ("", True),
+        ({}, True),
+        (None, False),
```

Comment:
Please change this to "null", because Yaml interprets only `null` as `None` when converted to Python, otherwise `None` interpolated in a string will be considered of type string.